docs/io/index_unofficial.rst

```diff
@@ -0,0 +1,287 @@
+their camera events are stored.
+
+For data analysis, MC verification, and increasing familiarity with using
+`ctapipe`, it is desirable to allow reading of all these data formats into
+`ctapipe`. However it is important to keep in mind the following points in the
+creation of a new unofficial reader:
+
+* These readers are unofficial and temporary - every camera team will be
+  adopting the official CTA data format once it is defined.
+* Maintaining all these readers is expensive and inefficient for the long-term.
+  Once cameras have transitioned to the new data format these unofficial
+  readers will be removed from `ctapipe`. Experts who wish to read the old data
+  formats are advised to either convert their old data into the official
+  format, or locally restore the unofficial `ctapipe` reader for that format.
+
+In an attempt to simplify and standardise the process of creating a new
+reader, this documentation has been created, containing the guidelines to be
+followed. These guidelines can be summarised as:
+
+* Each reader should have a corresponding
+  `ctapipe.io.eventfilereader.EventFileReader` in
+  `ctapipe.io.unofficial.eventfilereader` to ensure the common high-level
+  interface for reading data is maintained. This is explained in
+  `EventFileReaders`_.
+* Care should be taken in the imports of external modules. There should be no
+  dependencies added to ctapipe on the external libraries required for these
+  unofficial readers. The correct approach to achieve this is
+  explained in `EventFileReaders`_.
+
+
+How to use my reader in ctapipe?
+================================
+
+Presumably you already have some software that exists to read in your data
+format. What form that reader exists in dictates how easy it is to incorporate
+into ctapipe. Some of the possible forms and appropriate inplementations are:
+
+* **Your reader can be entirely programmed in Python**, in which case your
+  reader does not depend on any external packages, and you can include that
+  code in your reader's `Low-Level Reader-Specific Modules`_.
+* **Your reader exists as an external Python module**, in which case it could
+  be made available on conda for easy (optional) installation.
+* **You have a C/C++ inplementation of your reader in a simple script**,
+  therefore you could provide an interface to that functionality by using
+  "ctypes". The C/C++ scripts can exist inside your reader module, and are
+  compiled as part of the ctapipe installation. See
+  `ctapipe.utils.neighbour_sum` as an example of how to interface Python with
+  C code, and remember to add your extension to ext_modules in setup.py.
+* **Your reader exists as an external C/C++ library**, therefore software such
+  as SWIG can be used to provide a Python wrapper to the library,
+  automatically creating interfaces to the functions in the library.
+
+
+EventFileReaders
+================
+
+The module `ctapipe.io.unofficial.eventfilereader` contains the
```

Comment:
Thats not good!

There should not be a single file that contains `EventFileReaders` for different data formats.
I think this will be much easier to maintain if there are submodules for each data format instead of a single file mixing different file formats.

```python
from ctapipe.io.hessio import HessioEventFileReader
from ctapipe.io.zfits import ZfitsEventFileReader
```